_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "emldown: From machine readable EML metadata to a pretty documentation website"
```

Comment:
@maelle can you do a dash instead of a `:` - jekyll doesn't like those in titles